### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by Soitora**
**PR Changes**
- Removed `manifest.json`
- Organized assets into relative folders for their pages
- Deleted unused assets
- Small changes to some pages
- Removed unused items in `Navigation.vue`
- Keep sidebar from jumping when navigating, this was done to feature the active one on top but it became quite annoying.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <div>
-	<h1 align="center"><img src="./src/.vuepress/public/assets/media/tachiyomi.png">Tachiyomi Website</h1>
+	<span align="center"><img src="./src/.vuepress/public/icons/logo.png" height="100px"></span>
```
Comment (GHOSTBEAR) [inline README.md:2]: Image doesn't seem to be centered

File: src/help/faq/README.md [hunk 2]
```diff
@@ -129,10 +113,10 @@ Due to Covid-19 quarantine, most manga sites are under heavy load. Thus we do no
 Tachiyomi is not affiliated with or responsible for any source that is down or slow.
 :::
 
-<!--- 
+<!---
 While Tachiyomi does not officially endorse or recommend any manga website, below listed are some popular sources reported by users. **MangaDex**, **MangaSee** and **MangaNelo** are among the most popular in general, these are large sites which host content from scanlators. Description of some popular sources are listed below:
 -->
-<!--- 
+<!---
```
Comment (GHOSTBEAR) [inline src/help/faq/README.md:119]: Could this be cleaned up to be one large comment block?

File: README.md (hunk 1 above)
Comment (Soitora) [reply README.md:2]: image
This is for the GitHub README

File: src/help/faq/README.md (hunk 2 above)
Comment (Soitora) [reply src/help/faq/README.md]: Done in c08d2b0959b9fd5b7a97fe9e0a8ed418f2f226a3

File: README.md (hunk 1 above)
Comment (GHOSTBEAR) [reply README.md:2]: image
¯\_(ツ)_/¯

**Pull request merged**